a3_finance/a3_finance/doctype/asset_physical_verification/asset_physical_verification.py

```
@@ -3,7 +3,58 @@
+from erpnext.assets.doctype.asset.asset import create_asset_repair
 
 class AssetPhysicalVerification(Document):
 	pass
+
+import frappe
+
+def on_submit(doc, method):
+    for row in doc.asset:
+        if row.new_status == "Scrapped":
+            scrap_asset(row.asset)  # ✅ This triggers ERPNext's built-in scrapping process
+            
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)